tests/lib/openstack_query/query_blocks/test_query_grouper.py

```diff
@@ -0,0 +1,164 @@
+from unittest.mock import call, patch
+import pytest
+
+from openstack_query.query_blocks.query_grouper import QueryGrouper
+from enums.query.props.server_properties import ServerProperties
+
+from exceptions.parse_query_error import ParseQueryError
+
+
+from tests.lib.openstack_query.mocks.mocked_props import MockProperties
+
+
+@pytest.fixture(name="instance")
+def instance_fixture():
```

Comment:
Should this fixture have a scope?

Reply:
The scope is function - which is the default I believe - so I don't specify it